tests/testthat/test-get_quantile.R

```diff
@@ -1,14 +1,13 @@
-#' @title Specifications get_quantile
-#' @section Last updated by:
-#' Steven Haesendonckx
-#' @section Last update date:
-#' 24-MAY-2021
-
-# Specifications ----------------------------------------------------------
-#' T1.  The function accepts a survival object
+#' @title Specifications test-get_quantile.R
+#' @section Last updated by: Tim Treis (tim.treis(at)outlook.de)
+#' @section Last update date: 2021-10-28 16:29:24
+#'
+#' @section List of tested specifications
+#' T1. The function accepts a `survfit` object
 #' T1.1 No error when a `survfit` object is passed to the function with at least 2 strata
 #' T1.2 An error when a `survfit` object is passed to the function with 1 strata
-#' T1.3 An error when a non-`survfit` object is passed to the function
+#' testthat::test_that(An error when a non-`survfit` object is passed to the function
```

Comment:
?

Reply:
Love the watchdog already for exposing these things in the tests further down the file :D